### DISCUSSION TITLE
death-valley-2/

### DISCUSSION THREAD

**Discussion by giscus** [ironicbadger/blog.ktz.me#214] [Announcements]
# death-valley-2/

These desert vistas absolutely boggle my little mind. Most people come to Las Vegas for the brash, expensive and over the top spectacle. Indeed, it is fun for a night or two. But given the choice between a night in Vegas or another night on those dunes, there's simply no contest.

https://blog.ktz.me/death-valley-2/

**Comment by d4n3sh** [comment]
Impressed. I am interested in taking a solo trip now :)

**Reply by ironicbadger** [reply]
Do it! Being out there at midnight, truly alone like that makes one feel their mortality (if something happens to me, I'm screwed!) but in this modern world it's a rare feeling to be completely alone.

**Comment by charles-997** [comment]
Great pictures Alex.  I am also a hobbyist-photographer and have tried astro photography a few times.  I have yet to try it in a location as stunning as Death Valley.

The few times I have tried it- I noticed that composition is so challenging because I often could not see what the composition would be until after taking the image.

One thing I noticed is many of your images were shot at ISO 2500-3200.  I am really impressed with the results given I often expect a lot of noise in that range.  Goes to show that comparing ISO numbers across cameras is not a useful comparison, given clearly the R6 performs very well at those ISOs.  Assuming you had a tripod, (I am guessing you were not hand holding 15 second exposures!) what was your thinking around "only" a 15 sec exposure at 2500 vs lengthening exposure to get down to ISO 100? Clearly results were fantastic.

BTW my two favorites are the one with the last bit of light hitting the peaks & the road curving through the foreground and the image where you tried the light painting.  The stars are remarkable in that image.

**Reply by ironicbadger** [reply]
Like most things in life, the exposure / ISO / noise triangle is balance. If you go much longer than about 22 seconds the earth moves enough that the stars begin to look a little less than perfectly crisp. Stylistically that might be what you want. However using super low ISOs also increases risk of camera shake from wind or trees moving in the frame too. Again, it's a balance.

In another post from a few years ago in Utah I captured the Milky Way much more prominently (sadly the really colourful part was below the horizon this night according my ephemeris app). Scroll to the end of this post - https://blog.ktz.me/utahs-life-elevated - to see that one. That was shot on my older 6D vs the R6 used here - the noise performance on the R6 is noticeable better but then again, it is a 10 year newer camera so you'd expect that.

Another important difference between those images with the lens used. As I mentioned I used a cheap Samyang f/2.8 - a whole stop lighter than anything else wide I have. That allowed me to cut a few seconds off the exposure and get crisper stars without any trails.

Death Valley has to be one of the most challenging locations for any landscape photographer to shoot. Yes, there are some interesting features but they're framed by... nothingness. With the naked eye that's what makes them special but how do you convey that feeling through the composition without things getting lost in the vastness? I'm sure we've all taken panorama images on our phones and realised they are almost completely useless. Less is more, y'know?

And yes, you are right I used a tripod. This isn't an iPhone or Pixel phone with mad software processing - it's an old school detachable lens camera that requires a bit of effort.

Thank you for your kind words. I'm proud to say that the light painting image of the curly tree is almost entirely unedited and straight out the camera. Sometimes nature does all the hard work for you.

FYI - I will be putting together a small workshop trip with my good buddy Brent Gervais sometime in 2023 to visit Death Valley if you'd like to join us!

**Comment by pmeaney** [comment]
I think someone's account was hacked-- and the person who did it sent out a
malicious URL probably containing malware of a virus...
but that's just my assumption

Happy New Year!! :D
